tools/local-network-setup/README.md

```diff
@@ -54,10 +54,18 @@ You can specify to run anywhere between one and twenty nodes with the `--nodes`
 
 The first node will be named `bootstrap`, while subsequent nodes will be named `dh1, dh2, ...`. <br/>
 
+### MacOS
+
 ```bash
 bash ./tools/local-network-setup/setup-macos-environment.sh --nodes=6
```

Comment:
@aleksamagicka can you please put here and below that 12 is recommended number of nodes to start

Reply:
Done in https://github.com/OriginTrail/ot-node/pull/2922/commits/de5cc82b62fa96eb0de8d3be2d2f823093afb3e8.